pkg/azuredisk/nodeserver.go

```diff
@@ -272,13 +278,22 @@ func (d *Driver) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpublish
 	targetPath := req.GetTargetPath()
 	volumeID := req.GetVolumeId()
 
+	ephemeral := false
+	if match := managedDiskPathRE.FindString(volumeID); match == "" {
```

Comment:
In the persistent scenario, the volumeID is like `/subscriptions/xxx/resourceGroups/xxx/providers/Microsoft.Compute/disks/xxx`. But in the inline scenario, it is like `csi-fcfeb6b3905ea15047fd0a6a6c724049eae9e022ba0bd02c4af00feac400ac7e`

Reply:
I think you still need to format check on `volumeID` since it could be invalid format, and there is the format doc link of inline volume?

Reply:
Here is the generating process.
https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/csi/csi_mounter.go#L450-L454

The format totally complies with the azuredisk naming convention.

Reply:
at least check prefix `csi-`, and also add link in the comment, thanks

Reply:
done.